cfme/tests/generic_objects/test_instances.py

```diff
@@ -254,3 +254,23 @@ def test_generic_objects_tag_rest(appliance, generic_object, tags):
         generic_object.remove_tag(tag)
         tag_available = generic_object.get_tags()
         assert tag.id not in [t.id for t in tag_available]
+
+
+@pytest.mark.manual
+@pytest.mark.ignore_stream("5.10")
+def test_import_export_generic_object():
+    """
+    Bugzillas:
+        * 1595259
+
+    Polarion:
```

Comment:
Please add some test steps as this is a manual test and it's hard to know what to do for a person not familiar with this functionality

Reply:
@lina-nikiforova this is an upstream test, and as we don't have a build yet, I am unsure of the test steps myself. Perhaps I'm confused about the new procedure, do we wait until we have a build to merge these? I was under the impression that we want to make the test case now and then automate/add test steps later.

Reply:
@john-dupuy that's a valid point, however at least some general test steps would be useful as it will be updated later anyway.
Also, the test steps will be similar to these:
https://github.com/ManageIQ/manageiq/pull/18327
https://github.com/ManageIQ/manageiq/pull/18284

Reply:
Thanks @lina-nikiforova I will update this.